aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/AWSS3StoragePlugin.java

```diff
@@ -315,6 +313,10 @@ public AmazonS3Client getEscapeHatch() {
         return operation;
     }
 
+    private String getUserIdentityId() {
+        return identityIdProvider.getIdentityId();
```

Comment:
Maybe just inline this instead of using the getUserIdentityId()?

Reply:
That was how I initially had it, but I just force pushed this change because it looked ugly lol